vllm/v1/attention/backends/flash_attn.py

```diff
@@ -299,7 +302,9 @@ def __init__(self, runner: "GPUModelRunner"):
         self.num_heads_kv = model_config.get_num_kv_heads(
             runner.parallel_config)
         self.headdim = model_config.get_head_size()
-        self.page_size = self.runner.block_size
+        self.page_size = kv_cache_spec.block_size
```

Comment:
I think the term `page_size` is a bit confusing here. Maybe worth a comment (while it's currently the same as `block_size`).

Reply:
I've changed all `page_size` in this file to `block_size`.